src/model/AssetsCompiler.ts

```diff
@@ -317,21 +317,27 @@ export class AssetsCompiler {
     }
 
     private static findPluginPath(repositoryDir: string, pluginName: string, repoDependencies: string[]): string {
-        let relativePath = pluginName;
-        if (fs.existsSync(path.join(repositoryDir, relativePath))) {
-            return path.join(repositoryDir, relativePath);
+        let relativePathToPlugin = pluginName;
+        if (fs.existsSync(path.join(repositoryDir, relativePathToPlugin))) {
```

Comment:
We should do the same check that we do in parent repos also in the local repo, because otherwise we may get a false hit in the repo from which a plugin was moved out of.